docs/Splashkit/Applications/Arcade Machines/Guide to Contribute a Game/Contributors Guide.md

```diff
@@ -0,0 +1,152 @@
+# Guide to Contribute a Game to the Arcade Machine 
+
+### This guide takes you through the steps required for your game to be added into the arcade-machine library.  
```

Comment:
I would not put this as a heading, but that is my personal preference. I am happy to be over ruled